writeengine/splitter/we_filereadthread.cpp

```diff
@@ -463,12 +528,11 @@ void WEFileReadThread::openInFile()
                 cout << aGreenCol
                      << "trying to read from STDIN... "
                      << aDefCon << endl;
+             fInFile.rdbuf(cin.rdbuf());
```

Comment:
random bug I noticed in testing.  if you specified an input list like "file1 stdin", it wouldn't read from stdin.  This line fixes it for mode-1 imports.